src/js/components/Video/Video.js

```diff
@@ -296,6 +303,7 @@ const Video = forwardRef(
                     values={[{ value: percentagePlayed || 0 }]}
                   />
                   <StyledVideoScrubber
+                    aria-label={messages.scrubber || 'Scrubber'}
```

Comment:
Since we are providing a default value to messages.scrubber, I don't think there is a point using the OR operator here.